bowtie/schemas/cli/trend.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Tests? For this and generally?

Reply:
I am working on the code for accepting the `.tar` input file so we don't really have to make real **HTTP** calls for `bowtie trend` tests.